binapigen/plugin.go

```diff
@@ -14,38 +14,55 @@
 
 package binapigen
 
-import "fmt"
+import (
+	"fmt"
+	"plugin"
+	"strings"
+)
 
 type Plugin struct {
 	Name         string
-	GenerateFile func(*Generator, *File) *GenFile
+	GenerateFile func(*Generator)
```

Comment:
Should rather be named `Generate`, because we are dropping the `*File` parameter.